geomesa-convert/geomesa-convert-common/src/main/resources/META-INF/services/org.locationtech.geomesa.convert.TransformerFunctionFactory

```diff
@@ -3,4 +3,5 @@ org.locationtech.geomesa.convert.DateFunctionFactory
 org.locationtech.geomesa.convert.GeometryFunctionFactory
 org.locationtech.geomesa.convert.IdFunctionFactory
 org.locationtech.geomesa.convert.LineNumberFunctionFactory
-org.locationtech.geomesa.convert.MapListFunctionFactory
\ No newline at end of file
+org.locationtech.geomesa.convert.MapListFunctionFactory
+org.locationtech.geomesa.convert.CastFunctionFactory
```

Comment:
Nothing should have changed on line 6, not sure what is going on there.  This is a (text file?) list of factories and not imports, so not sure if there is any cleanup I can do here.